team-practices.md

```diff
@@ -12,11 +12,30 @@ with our 18F team, and daily standups with the DOJ team.
 We use Github + Zenhub for issue tracking. We create issues as necessary to capture stories
 and articulate requirements for specific tasks. Our kanban categories for describing the state of
 issues are:
- * New
- * Backlog
- * In Progress
- * Review / QA
- * Closed
+
+#### New
+
+Default for created issues.
+
+#### Icebox
+
+Has not yet been groomed.
```

Comment:
Icebox'd stories are low priority things, they might have been groomed but then we decided we're not going to be working on them this sprint. From Trello:

> The Icebox is where items go that have lower priority or we are unsure of their state. This tends to be the dumping grounds of things we'd like to do but there are many other things with higher priority.